modules/vmware-csi-driver-reqs.adoc

```diff
@@ -21,3 +21,133 @@ If a third-party vSphere CSI driver is present in the cluster, {product-title} d
 ====
 The VMware vSphere CSI Driver Operator is supported only on clusters deployed with `platform: vsphere` in the installation manifest.
 ====
+
+You can create a custom role for the Container Storage Interface (CSI) and the CSI Driver Operator. The custom role can include privilege sets that assign a minimum set of permissions to each vSphere object. This means that the Container Storage Interface (CSI) and CSI Driver Operator can establish a basic interaction with these objects. The following tables detail these minimum set of permissions.
+
+[IMPORTANT]
+====
+Installing an {product-title} cluster in a vCenter is tested against a full list of privileges as described in the "Required vCenter account privileges" section. By adhering to the full list of privileges, you can reduce the possibility of unexpected and unsupported behaviors that might occur when creating a custom role with a set of restricted privileges. 
+====
+
+.Minimum permissions for the CSI
+[%collapsible]
+====
+[cols="4a,4a,3a",options="header"]
+|===
+|vSphere object for role
+|When required
+|Required privileges
+
+|vSphere vCenter
+|Always
+|
+[%hardbreaks]
+`Cns.Searchable`
+`Read Only`
```

Comment:
Hi @dfitzmau, I might miss something, but if we need some privileges, why do we need "Read Only" here? I think "Read Only" is only applicable for no specific privilege needed like "vSphere Port Group" in Line 74.

Reply:
Thanks, @duanwei33 . @rvanderp3 , if a vSphere object requires minimum permissions that extends beyond `Read Only`, do we need to list `Read Only`?

Reply:
Hi @duanwei33 . Updated. Does the content LGTY?